### PR TITLE
feat: remove hero from mobile

### PR DESCRIPTION
**Pull request by ahmaxed**

### CODE REVIEW

File: post.hbs [hunk 1]
```diff
@@ -49,13 +49,17 @@ into the {body} of the default.hbs template --}}
...
-                <img srcset="{{img_url feature_image size="s"}} 300w,
-                            {{img_url feature_image size="m"}} 600w,
-                            {{img_url feature_image size="l"}} 1000w,
-                            {{img_url feature_image size="xl"}} 2000w" sizes="(max-width: 800px) 400px,
-                            (max-width: 1170px) 700px,
-                            1400px" src="{{img_url feature_image size="xl"}}" alt="{{title}}"
-                    onerror="this.style.display='none'" />
+                <picture>
+                    <source media="(max-width: 700px)" sizes="1px"
+                        srcset="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7 1w" />
```
Comment (ojeytonwilliams) [inline post.hbs:54]:
```suggestion
```
If this source is missing, nothing gets loaded for small screens which is what we want.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: post.hbs (hunk 1 above)
Comment (ahmaxed) [reply post.hbs:54]: seems like it will load the <img src which is our xl image

Comment (ojeytonwilliams) [reply post.hbs:54]: Thanks, I understand now - it worked locally only because the src was missing.  Now that you've fixed that, it defaults to it.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**